middleware/auth-utils/token.js

```diff
@@ -39,6 +39,7 @@ function Token (token, clientId) {
       this.signature = new Buffer(parts[2], 'base64');
       this.signed = parts[0] + '.' + parts[1];
     } catch (err) {
+      console.error('Failed to parse JWT token', err);
```

Comment:
could  we use log instead of console?

Reply:
I did not want to introduce any additional dependencies in this package. Let's discuss

Reply:
Oh... I forgot that it isn't our own library... never mind